packages/app-mobile/ios/Joplin/Info.plist

```diff
@@ -39,6 +39,13 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
+		<!-- Do not change NSAllowsArbitraryLoads to true, or you will risk app rejection! -->
+		<key>NSAllowsArbitraryLoads</key>
+		<false/>
+		<key>NSAllowsLocalNetworking</key>
+		<true/>
```

Comment:
This change is from the upgrade diff. However, it **may** address #10420 (untested).

Reply:
IOS17.2.1 + Joplin 12.14.8 code + this modify(NSAllowsLocalNetworking), not fix #10420  .
I found a alternatives way: #10437

Reply:
Not sure there's anything we can do here since we can only whitelist specific hosts? Or maybe we could whitelist a generic host like `sync.server` and document this somewhere? Then the user can do their own setup and use this hostname to get it working